### ISSUE TITLE
merge our vcr wrapper and the one from @pulp/squeezer

### ISSUE THREAD

**Issue by evgeni** [theforeman/foreman-ansible-modules#883]
they have common history and probably should be split out into a `ansible-vcr-wrapper` project that both can use.

**Comment by evgeni** [comment]
cc @mdellweg

**Comment by mdellweg** [comment]
The same holds true for the Makefile.
Maybe we can accompany this with some ansible collection specific makefile macros (*.mk files).

**Comment by evgeni** [comment]
True. But distributing/including these will be a bit more complicated, as the Makefile would need to find out the path contrary to just importing a module
